src/plugins/intel_gpu/src/kernel_selector/cl_kernels/mvn_gpu_bfyx_opt.cl

```diff
@@ -4,45 +4,57 @@
 
 #include "include/batch_headers/fetch_data.cl"
 
+#if !IS_DYNAMIC
 __attribute__((reqd_work_group_size(LWS, 1, 1)))
+#endif
 KERNEL (mvn_gpu_bfyx_opt)(
+    OPTIONAL_SHAPE_INFO_ARG
     const __global INPUT0_TYPE* input,
     __global OUTPUT_TYPE* restrict output
 #if HAS_FUSED_OPS_DECLS
     , FUSED_OPS_DECLS
 #endif
 ) {
-    const uint data_set_idx = get_global_id(1);     //in processing of which data set this WI participates?
-    const uint workers_per_data_set = LWS;          //how many WI participates in processing of one data set
-    const uint in_data_set_idx = get_global_id(0);  //this WI's id in group of items processing single data set
-    const uint data_set_size = DATA_SET_SIZE;       //how many elements are in one data set
-    const uint data_sets_count = DATA_SETS_COUNT;   //how many data sets are in the processing payload
+    const uint data_set_idx = get_global_id(1);                   // in processing of which data set this WI participates?
+    const uint workers_per_data_set = get_local_size(0);          // how many WI participates in processing of one data set
```

Comment:
This is power of two and being used in multiplicaiton & div & modular in many times. Could we change them all to bit shift? 
e.g, workers_per_data_set = 8 / bits  = 3 
i *workers_per_data_set => i << 3
i / workers_per_data_set => i >> 3 
i % workers_per_data_set => i - (i >> 3) << 3

Reply:
updated to use built-in log2 function and bit shift as your suggestion